omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -240,55 +240,60 @@ function PatientSearchWidget(configuration){
 
     var updateSearchResults = function(results){
         var dataRows = [];
-        if(results){
-            var results = removeDuplicates(results);
-            searchResultsData = searchResultsData.concat(results);
-            _.each(results, function(patient) {
-                var birthdate = '';
-                var widgetBirthdate = patient.person.birthdate;
-                if(patient.person.birthdate){
-                    birthdate = moment(patient.person.birthdate).format(configuration.dateFormat);
-                    if( patient.person.birthdateEstimated ){
-                        birthdate = "~ "+birthdate;
-                    }else{
-                        birthdate = "&nbsp;&nbsp; "+birthdate;
-                    }
-                }
-                var identifier = patient.patientIdentifier != null ? patient.patientIdentifier.identifier : null;
-                if(_.contains(initialPatientUuids, patient.uuid)){
-                    identifier = patient.patientIdentifier.identifier+
-                        " <span class='recent-lozenge'>"+config.messages.recent+"</span>";
-                }
-                var age = patient.person.age;
-                if(age == '' && widgetBirthdate != ''){
-                    age = formatAge(widgetBirthdate);
+        var results = removeDuplicates(results);
+        searchResultsData = searchResultsData.concat(results);
+        _.each(results, function(patient) {
+        	var birthdate = '';
+            var widgetBirthdate = patient.person.birthdate;
+            if(patient.person.birthdate){
+            	birthdate = moment(patient.person.birthdate).format(configuration.dateFormat);
+                if( patient.person.birthdateEstimated ){
+                	birthdate = "~ "+birthdate;
```

Comment:
It would be better if we have some space between `"~" + birthdate`

Reply:
i actually didn't touch this line, but will make the space